nano/templates/chem_heater.tmpl

```diff
@@ -33,7 +41,7 @@ Used In File(s): \code\modules\reagents\newchem\chem_heater.dm
 				<div style="width: 100%;">
 					{{if data.isBeakerLoaded}}
 						<b>Volume:&nbsp;{{:helper.smoothRound(data.beakerCurrentVolume)}}&nbsp;/&nbsp;{{:data.beakerMaxVolume}}</b><br>
-						<b>Temperature:&nbsp;{{:helper.smoothRound(data.currentTemp)}}&nbsp;Kelvin</b><br>
+						<b>Temperature:&nbsp;<span class="{{:data.targetTempReached}}">{{:helper.smoothRound(data.currentTemp)}}</span>&nbsp;Kelvin</b><br>
```

Comment:
This makes it harder to see what is happening here at a glance.
Best to make this an if statement with a boolean value instead of the classname being passed by the code behind.

Reply:
I've added newlines and replaced the class name by a ternary, resulting in this:
```
<b>
	Temperature:&nbsp;
	<span class="{{:data.targetTempReacheddata.targetTempReached ? 'good' : 'average'}}">{{:helper.smoothRound(data.currentTemp)}}</span>&nbsp;Kelvin
</b><br>
```

I'm not sure if this is what you meant by an if statement or you really want me to use one. If it's the latter, I'm not sure it's good because it would result in duplicate code.

Reply:
Aye this is a good way to solve it. Looks good!